events/2019/11-contributor-summit/RETRO.md

```diff
@@ -0,0 +1,267 @@
+# <p style="text-align: right">
+San Diego Contributor Summit 2019 - Retro</p>
+
+
+<p style="text-align: right">
+status: <em>in progress</em></p>
+
+
+<p style="text-align: right">
+date:  2019-11-06</p>
+
+
+<p style="text-align: right">
+<em>author: jeefy + paris</em></p>
```

Comment:
I know this is taking the raw google doc, but I'd scrap this section and just leave it as:
```
# San Diego Contributor Summit 2019 - Retro

date:  2019-11-06

author: jeefy + paris

```